internal/fs/local_file_test.go

```diff
@@ -736,3 +735,56 @@ func (t *LocalFileTest) TestRmDirOfDirectoryContainingOnlyGCSFiles() {
 	t.validateObjectNotFoundErr("explicit/foo")
 	t.validateObjectNotFoundErr("explicit/bar")
 }
+
+func (t *LocalFileTest) TestCreateSymlinkForLocalFile() {
+	var filePath string
+	// Create a local file.
+	filePath, t.f1 = t.createLocalFile(FileName)
```

Comment:
should we close the open file handle in t.f1 ?

Reply:
done.